src/pages/organisationUnits/list/OrganisationUnitList.tsx

```diff
@@ -219,7 +220,10 @@ export const OrganisationUnitList = () => {
     return (
         <div>
             <SectionListTitle />
-            <IdentifiableFilter />
+            <div className={css.filterWrapper}>
+                <IdentifiableFilter />
+                <DynamicFilters />
```

Comment:
Do we actually want additional filters for orgunits? It does make it a bit weird. eg. if we had a filter to filter on a certain level - how do we display the tree?

Reply:
ah that's a good point. Probably not right? prob best to remove filters from public view all together?

Reply:
ok @Birkbjo i just removed the filter tabs all together